app/views/applications/timeline/show.njk

```diff
@@ -16,6 +16,38 @@
     <div class="govuk-grid-column-two-thirds">
 
       <h2 class="govuk-heading-l govuk-!-margin-bottom govuk-!-font-size-36">Timeline</h2>
+      {# {% if application.status == 'Offered'%}
+        <h3 class="govuk-heading-s govuk-!-font-size-24 govuk-!-margin-bottom-1">Application updated</h3>
+        <p class="app-activity-log__byline">
+          Candidate,<time datetime="2023-07-09t09:01:54.251+01:00">2023-07-31T12:01:54.251+01:00</time>
+        </p>
+        <div class="app-activity-log__item">
+          <div class="app-activity-log__box">
+            {{ govukSummaryList({
+  rows: [
+    {
+      key: {
+        text: "Phone number"
+      },
+      value: {
+        text: "078536374"
+      }
+    },
+    {
+      key: {
+        text: "Address"
+      },
+      value: {
+        html: "72 Guild Street<br>London<br>SE23 6FH"
+      }
+    }
+  ]
+}) }}</div>
+          <p class="app-log-card__note--link">
+            <a href="/applications/{{ application.id}}">View changes</a>
```

Comment:
For the link 'View changes' I wonder if we should just say 'View application'. I'm thinking if they update more than 1 section, like 'Personal details' and then 'Equality and diversity' which appear at different places on the form then which do we link to. So maybe we just keep it general and link to the overall application tab. I think the design here is clear enough on whats changed anyway!